### ISSUE TITLE
auto_compaction option does not work?

### ISSUE THREAD

**Issue by ssured** [pouchdb/pouchdb#3345]
Following the API from http://pouchdb.com/api.html#create_database, it seems passing `auto_compaction: true` in the constructor does not work.

See this fiddle http://jsfiddle.net/2j2ap9mb/ for a minimal implementation on PouchDB 3.2

It might be that the `.info()` is broken, but I did not check that.

**Comment by nolanlawson** [comment]
Little typo in the code. I have a fix.

**Comment by ssured** [comment]
Cool! One more thing to add to 3.2.1? Thanks!

**Comment by ssured** [comment]
I reviewed your code and it appeared the issue was in `.info()`, which in the end is a 'cosmetic' change. I wonder if there is any way to check that auto_compaction runs? For my app it is quite important that space is freed up when some docs (with several mb of attachments) are removed, it would be nice to see that happen in some way.

**Comment by nolanlawson** [comment]
Yes, that is tested pretty heavily in `test.compaction.js`. The way to be sure is to gather all the revisions and check that non-leaf revisions are unavailable. In [the compaction guide](http://pouchdb.com/guides/compact-and-destroy.html) there are a few live examples that demonstrate how this works.
